brody_lab_to_nwb/convert_brody/brodynwbconverter.py

```diff
@@ -1,21 +1,25 @@
 """Authors: Cody Baker."""
-from nwb_conversion_tools import NWBConverter, SpikeGadgetsRecordingInterface, NeuralynxRecordingInterface, \
-    SpikeGLXRecordingInterface, SpikeGLXLFPInterface
+from nwb_conversion_tools import NWBConverter, NeuralynxRecordingInterface, SpikeGLXRecordingInterface, \
+    SpikeGLXLFPInterface
 
-from .brodybehaviordatainterface import BrodyBehaviorDataInterface
+from .poissonclicksdatainterface import PoissonClicksDataInterface
+from .neuralynxbehaviordatainterface import NeuralynxBehaviorDataInterface
 
 
-class BrodysNWBConverter(NWBConverter):
-    """Primary conversion class for the Brody lab processing pipeline."""
+class PoissonClicksNWBConverter(NWBConverter):
+    """Primary conversion class for the SpikeGLX formatted Brody lab data."""
 
     data_interface_classes = dict(
-        SpikeGadgetsRecording=SpikeGadgetsRecordingInterface,
-        NeuralynxRecording=NeuralynxRecordingInterface,
         SpikeGLXRecording=SpikeGLXRecordingInterface,
         SpikeGLXLFP=SpikeGLXLFPInterface,
```

Comment:
@CodyCBakerPhD are there separate files for raw and lfp in this case?

Reply:
Yep. Standard SpikeGLX output and actively choosing to write both here.

Reply:
I think [these changes](https://github.com/catalystneuro/nwb-conversion-tools/pull/189) will be required for this to work

Reply:
see also https://github.com/catalystneuro/movshon-lab-to-nwb/blob/c1cac6426a6fc6c31b7eee7d8a9bb23281bc8331/movshon_lab_to_nwb/movshonnwbconverter.py#L43-L48